code/modules/surgery/organs/stomach.dm

```diff
@@ -16,11 +16,15 @@
 	high_threshold_cleared = "<span class='info'>The pain in your stomach dies down for now, but food still seems unappealing.</span>"
 	low_threshold_cleared = "<span class='info'>The last bouts of pain in your stomach have died out.</span>"
 
+	//manages it's own reagents, will inject food reagents on first bite.
+	food_volume = 0
+
 	var/disgust_metabolism = 1
 
 /obj/item/organ/stomach/Initialize()
 	. = ..()
 	create_reagents(1000)
+	RegisterSignal(src, COMSIG_FOOD_EATEN, .proc/on_bite)
```

Comment:
this is what callbacks are for

Reply:
False,

the callbacks are after_eat, on_consume
Both happen after reagents are transfered.

Reply:
I didn't say thats what the existing callbacks are for. I'm saying this use-case is what asks for a callback. not a signal.